lib/pharos/configuration/host.rb

```diff
@@ -20,6 +20,7 @@ class Host < Dry::Struct
       attribute :http_proxy, Pharos::Types::Strict::String
 
       attr_accessor :os_release, :cpu_arch, :hostname, :api_endpoint, :private_interface_address, :checks
+      attr_accessor :resolv_localhost, :systemd_resolved_stub
```

Comment:
probably should be under `checks`?

Reply:
Dunno, it's just as much metadata as `os_reselase` and `cpu_arch` are? It's not a healthcheck...

Reply:
ok, maybe then an accessor that has both attributes?

Reply:
A new `class Pharos::Configuration::HostResolver < Dry::Struct`?

Reply:
Alternatively, we could do these checks directly in the `ConfigureKubelet` phase, and skip the host attributes... they are not necessary in this case.

Reply:
A new class sounds good to me.